src/maintainer/conda_forge_yml.rst

```diff
@@ -291,6 +291,15 @@ modified.  Tools like conda-smithy may modify this, as need.  It has a single
       secure:
         BINSTAR_TOKEN: <some big hash>
 
+upload_on_branch
+----------------
+This parameter restricts uploading access on work from certain branches of the same repo.
```

Comment:
What about: 
> Only branches listed in `upload_on_branch` will build and upload packages on the conda-forge channel.

Reply:
The building is done for any branch, but uploading will only happen for specific branch(es). Also, `conda-forge` channel may not be true for a third-party org.